### PR TITLE
More layout bug fixes

### PR DESCRIPTION
**Pull request by matthewcoufal**
Ignore the expo settings and packager-info json files. Accidentally created those when trying to run a different project in this directory - I just deleted them.

- “Join” button text should be 50% opacity when button is disabled
- When displaying latest activity time stamp, currently we calculate # days ago by hours, but a better way would be to compare the calendar date of the last activity to todays date. 
- Overflow card is a different length for multi-choice VS free response, should be same length

### CODE REVIEW

File: Clicker/Quarks/DraftQuark.swift [hunk 1]
```diff
@@ -106,7 +106,6 @@ struct UpdateDraft: ClickerQuark {
     }
     var parameters: Parameters {
         return [
-            "id": id,
```
Comment (kevinchan159) [inline Clicker/Quarks/DraftQuark.swift]: why are we removing this?

File: Clicker/SectionControllers/CardSectionControllers/FRInputSectionController 2.swift [hunk 2]
```diff
@@ -0,0 +1,54 @@
+//
```
Comment (kevinchan159) [inline Clicker/SectionControllers/CardSectionControllers/FRInputSectionController 2.swift]: is this a duplicate file?

File: Clicker/SectionControllers/CardSectionControllers/HamburgerCardSectionController 2.swift [hunk 3]
```diff
@@ -0,0 +1,34 @@
+//
```
Comment (kevinchan159) [inline Clicker/SectionControllers/CardSectionControllers/HamburgerCardSectionController 2.swift]: duplicate file?

File: Clicker/SectionControllers/PollOptionsCellSectionControllers/FROptionSectionController.swift [hunk 4]
```diff
@@ -36,8 +36,7 @@ class FROptionSectionController: ListSectionController {
         guard let containerSize = collectionContext?.containerSize else {
             return .zero
         }
-        let cellHeight = calculateFROptionCellHeight(for: frOptionModel)
-        return CGSize(width: containerSize.width, height: cellHeight)
+        return CGSize(width: containerSize.width, height: LayoutConstants.frOptionCellHeight)
```
Comment (kevinchan159) [inline Clicker/SectionControllers/PollOptionsCellSectionControllers/FROptionSectionController.swift]: Is `frOptionCellHeight` going to be the same as the cell heights for the multiple choice cells?

File: Clicker/Quarks/DraftQuark.swift (hunk 1 above)
Comment (matthewcoufal) [reply Clicker/Quarks/DraftQuark.swift]: Yuna told me that we don't need to specify the id in that parameters variable, that the request will still work without it.

**Pull request merged**